spki/src/spki.rs

```diff
@@ -37,23 +37,23 @@ pub type SubjectPublicKeyInfoRef<'a> = SubjectPublicKeyInfo<'a, AnyRef<'a>>;
 /// ```
 ///
 /// [RFC 5280 § 4.1.2.7]: https://tools.ietf.org/html/rfc5280#section-4.1.2.7
-#[derive(Copy, Clone, Debug, Eq, PartialEq)]
-pub struct SubjectPublicKeyInfo<'a, Params> {
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct SubjectPublicKeyInfo<Params> {
```

Comment:
This obviously doesn't work as this, the intention was to remove the lifetime from here, maybe I should add a parameter like 
```suggestion
pub struct SubjectPublicKeyInfo<Params,Pkey> {
```

And consume that as a 
```
SubjectPublicKeyInfo<Any,BitString>  // Owned version
SubjectPublicKeyInfo<AnyRef<'a>,BitStringRef<'a>>  // Owned version
```

Reply:
It definitely needs a generic parameter so as to continue supporting a heapless profile.

I was thinking of using a generic `Bits` parameter with `&[u8]` and `Box<[u8]>` type aliased as you propose. The former could probably be called `type SubjectPublicKeyInfoOwned`.

There's a potential argument to be made that using a `BitString*` type is semantically more correct, although I'm not aware of any public key formats which aren't octet-aligned.

Reply:
I went with `BitString`/`BitStringRef` as to stick as close as possible to the RFC definition.
I can reuse the `Bytes` type I introduced in #771